db/seeds/hhg_weights.json

```diff
@@ -694,10 +666,6 @@
         "weight": 420
       },
       {
-        "name": "Organ (Medium)",
```

Comment:
Not enough servicemembers have an organ to justify this, huh?

Reply:
Yeah seems like a rare item